maidroid/script/instruction_set_doc.lua

```diff
@@ -1,22 +1,29 @@
 -- See also the pdisc mod's instruction list.
 
 local instr = {
-	{"getpos", "<s posname>", 'Gets the current position. (Stored in {x=vars[posname .. ".x"], y=[…]})'},
-	{"getvelocity", "<s velname>", "Gets the maidroid's velocity."},
-	{"getacceleration", "<s accname>", "Gets the maidroid's acceleration."},
-	{"getyaw", "<v yaw>", "Gets the maidroid's yaw."},
+	{"get_pos", "<s posname>", 'Gets the current position. (Stored in {x=vars[posname .. ".x"], y=[…]})'},
+	{"get_velocity", "<s velname>", "Gets the maidroid's velocity."},
+	{"get_acceleration", "<s accname>", "Gets the maidroid's acceleration."},
+	{"get_yaw", "<v yaw>", "Gets the maidroid's yaw."},
 
 	{"get_node", "<s posname>", 'Gets the node at posname, name, param2 etc. are stored in vars[posname .. ".name" etc.]. The maximum range depends on the tool.'},
 	{"get_nodedef", "<vs nodename>, <vs deftblname>", "Copies some fields of nodename's node definition, e.g. ${deftblname}.walkable := minetest.registered_nodes[nodename].walkable. If the node is unknown, nodename is set to false. For a list of supported fields, look at maidroid_core/cores/ocr.lua."},
 	{"get_item_group", "<vs name>, <s group>", "Sets name to the minetest.get_item_group(name, group)."},
 
-	{"setyaw", "<n yaw>", "Sets the maidroid's yaw in radians."},
-
+	{"set_yaw", "<n yaw>", "Sets the maidroid's yaw in radians."},
 	{"jump", "[<n heigth>]", "Makes the droid jump, if height is invalid (height ∈ ]0,2]), it's set to 1, if it's a variable, it's set to a bool indicating whether the jump succeeded."},
 	{"setwalk", "[<n speed>]", "Starts walking n m/s to the current direction. If speed is omitted, the maidroid stops. maximum speed: 5"},
+
 	{"dig", "<s posname>[, <v errormsg>]", "Digs a node, if posname is a variable, it is set to a bool indicating whether digging succeeded. If false, errormsg tells the reason."},
-	{"place", "<s posname>[, <v errormsg>]", "Places a node, if posname is a variable, it is set to a bool indicating whether placing succeeded. If false, errormsg tells the reason."},
+	{"place", "<s posname>[, <v errormsg>]", "Places a node, if posname is a variable, it is set to a bool indicating whether placing succeeded. If false, errormsg tells the reason. Can call on_rightclick instead if given."},
+	{"rightclick", "<s posname>[, <v errormsg>]", "Calls object's or node's on_rightclick. Can instead place one node nearer to droid if no on_rightclick in node definition."},
+	{"use", "[<s posname> | <n object_id>]", "Calls wielded item's on_use. If no on_use, punches."},
```

Comment:
bei use gibt es beim Spieler keine position, wenn das item kein on_use hat solls einfach nix machen (so wie wenn der spieler in den Himmel schaut)